doc/common/apidoc/index.json

```diff
@@ -8102,7 +8102,15 @@
                             "readonly": false,
                             "type": "\"input\" | \"button\"",
                             "default": "'button'",
-                            "description": "Icon position of the component. This only applies if the showIcon option is set to true."
+                            "description": "Visual style for the icon. This only applies if the showIcon option is set to true."
```

Comment:
(2) The documentation seemed misleading as talk about the icon's position instead of the display style.